src/server/req_runjob.c

```diff
@@ -1414,7 +1425,7 @@ post_sendmom(struct work_task *pwt)
 				 * array starts its comment is set to a begun message and
 				 * should not change after that
 				 */
-				if (jobp->ji_parentaj->ji_qs.ji_state != JOB_STATE_BEGUN) {
+				if (jobp->ji_parentaj->ji_qs.ji_state == JOB_STATE_QUEUED) {
```

Comment:
Please modify the comment above

Reply:
Addressed in https://github.com/PBSPro/pbspro/pull/1119/commits/2d3a5a6fe3c51e0f88bb63f19c7624fbca4a48d0